rhino-tools/src/main/java/org/mozilla/javascript/tools/shell/Main.java

```diff
@@ -551,7 +551,7 @@ static void processFileSecure(Context cx, Scriptable scope, String path, Object
         Object source = readFileOrUrl(path, !isClass);
 
         byte[] digest = getDigest(source);
-        String key = path + "_" + cx.getOptimizationLevel();
+        String key = path + "_" + (cx.isInterpretedMode() ? "int" : "comp");
```

Comment:
'int' might be confusing, is there any good reason for this shortening?

Reply:
Not really -- AFAIK this is just used as a cache key but there's no reason why not to make it longer. (Previously it was just an integer.)

Reply:
Checkout the latest patch, it supports both "-int" and "-interpreted"